lib/ansible/modules/network/fortios/fortios_cmdb_set.py

```diff
@@ -0,0 +1,728 @@
+         webfilter override-user, webfilter profile, webfilter search-engine, \
+         webfilter urlfilter, wireless-controller ap-status, \
+         wireless-controller global, wireless-controller setting, \
+         wireless-controller timers, wireless-controller vap, \
+         wireless-controller vap-group, wireless-controller wids-profile, \
+         wireless-controller wtp, wireless-controller wtp-group, \
+         wireless-controller wtp-profile]
+
+    mkey:
+        description:
+            - Master key of the resource for those endpoints \
+            representing a table
+
+    config_parameters:
+        description:
+            - Any of the additional parameters that are required by some \
+            endpoints to complete the request
+        required: true
+'''
+
+EXAMPLES = '''
+- hosts: localhost
+  vars:
+   host: "192.168.40.8"
+   username: "admin"
+   password: ""
+   vdom: "root"
+  tasks:
+  - name: Set static route on the fortigate
+    fortios_cmdb_set:
+     host:  "{{  host }}"
+     username: "{{  username}}"
+     password: "{{ password }}"
+     vdom:  "{{  vdom }}"
+     endpoint: "router static"
+     config_parameters:
+       seq-num: "8"
+       dst: "10.10.32.0 255.255.255.0"
+       device: "port2"
+       gateway: "192.168.40.252"
+  - name:   firewall policy
+    fortios_cmdb_set:
+     endpoint: "firewall policy"
+     host: "{{ host }}"
+     username: "{{ username }}"
+     password: "{{ password }}"
+     vdom:  "{{  vdom }}"
+     config_parameters:
+        policyid: "66"
+        name: "ansible"
+        json:
+          policyid: "66"
+          name: "ansible"
+          action: "accept"
+          srcintf: [ {"name": "port1"} ]
+          dstintf: [{"name":"port2"} ]
+          srcaddr: [{"name":"all"} ]
+          dstaddr: [{"name":"all"}]
+          schedule: "always"
+          service:  [{"name":"HTTPS"}]
+          logtraffic: "all"
+'''
+
+RETURN = '''
+results:
+  description: Data returned by the endpoint operation
+  returned: on sucess
+  type: string
+  sample: {
+            "apply-to": "admin-password ipsec-preshared-key",
+            "change-4-characters": "disable",
+            "expire-day": 90,
+            "expire-status": "enable",
+            "min-lower-case-letter": 2,
+            "min-non-alphanumeric": 1,
+            "min-number": 2,
+            "min-upper-case-letter": 2,
+            "minimum-length": 8,
+            "reuse-password": "disable",
+            "status": "enable
+          }
+
+status:
+  description: Indication of the operation's result
+  returned: always
+  type: string
+  sample: "success"
+
+version:
+  description: Version of the FortiGate
+  returned: always
+  type: string
+  sample: v5.6.2
+
+'''
+
+from ansible.module_utils.basic import AnsibleModule
+import logging
+
+try:
+    from fortiosapi import FortiOSAPI
+    HAS_FORTIOSAPI = True
+except ImportError:
+    HAS_FORTIOSAPI = False
+
+if HAS_FORTIOSAPI:
```

Comment:
Please delete this section

Reply:
Thanks gundalow, seems to work now. I'll submit a new commit shortly.